app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,43 @@
+class CommentsController < ApplicationController
+  def create
+    @talk = Talk.find(params[:talk_id])
+    @comment = Comment.new(talk_id: @talk.id, content: params[:comment][:content], user_id: 1)
+    if @comment.save
+      flash[:notice] = 'Thanks for commenting'
+    else
+    @comment.errors.full_messages.each do |error|
```

Comment:
Unused block argument - `error`. You can omit the argument if you don't care about it.<br>Use 2 (not 0) spaces for indentation.